README.md

```diff
@@ -1,61 +1,122 @@
-* Exercise consideration and respect in your speech and actions. Tone on the internet is often misunderstood, and people may interpret your words differently than you might intend. So err on the side of caution.
+> Be curious to hear about the other person’s perspective
+
+Remember not to assume your experience is the default - make space for others to have different life experiences than you. One litmus test for how you are framing something is going to read as having implicit bias is to check against these three questions: 
+* *First: is this true* is what you're about to say something that is a fact, or a perspective that you hold where you can acknolwedge that others may hold different ones? Is there a way to explicitly acknolwedge that you're open to other perpsectives in how you're responding; will this read as a "yes and", or a "no because"? 
+* *Second: Is this missing something* are you encouraging all possibilities, or just the ones that come most readily to you and your life experience? What questions might you ask the other folks in the conversation to understand their experiences or motivations better? 
+* *Third: is this just* is this supporting the inclusion and equality of minority members of our community, or reinforcing barriers for those groups? This can be quite nuanced, but being explicit about who and how we want to advocate for each other, and why it matters to us, is always a great reminder. 
+
+Take the time to understand the motivations behind what the other person has share, rather than drawing your own conclusions. When in doubt: repeat back what you think you're hearing, and ask why the topic matters to them. 
+
+> We all have to help to foster safer and braver spaces
+
+Different opinions are welcome in YYJ Tech, but context matters greatly. Please do your best to read the room and not derail the purpose(s) of the channels you are in, and respect the boundaries of others if they ask something of you. 
+
+Figuring out how to include someone who does not have the same experience as you is not easy, and if we succeed in being a welcoming place that will mean we are interacting with many more people with many diverse experiences. Don't be afraid to try, but be open to feedback and don't take offense if someone asks you to do something differently to include them. Work together to find a way to make it possible. 
+
+**The following behaviors are expected and requested of all community members:**
+
+* Participate in a respecful and active way that supports these community values above. In doing so, you contribute to the health and longevity of this community.
+* Exercise self-reflection and empathy in your speech and actions. Tone on the internet is often misunderstood, and people may interpret your words differently than you might intend. So err on the side of caution, and include the why behind your thoughts as often as possible.
```

Comment:
by `the why behind your thoughts` are we talking `the reasoning behind your thoughts` or something else?